src/views/dialogs/DepositDialog2/DepositForm.tsx

```diff
@@ -52,21 +75,28 @@ export const DepositForm = ({
           onSelectSpeed={setSelectedSpeed}
         />
       )}
+      {/* TODO(deposit2.0): make this error message better */}
+      {error && (
+        <div tw="text-center">
+          There was an error. Please increase your deposit amount and try again.
+        </div>
```

Comment:
do you think we should use `<AlertMessage>` component? Or does it stray too far from your ideal design

Reply:
gonna amend this in a bit actually, dan gave me new designs

Reply:
done now!